README.md

```
@@ -41,16 +41,23 @@ fn main() {
 ## Usage as a library
 
 To use this library in your project, enter your project directory and run:
-```bash
+```none
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)